packages/cli/src/middleware/index.js

```diff
@@ -1,60 +1,24 @@
-import { getTruffleConfig } from '../helpers/truffle-config'
```

Comment:
The middleware logic to setup an environment is not available on aragonCLI anymore and is all handle by the toolkit